shared/profile/index.desktop.js

```diff
@@ -343,31 +343,52 @@ class ProfileRender extends PureComponent<Props, State> {
                 )}
               </Box>
               <Box style={styleProofs}>
-                {!loading &&
-                  this.props.userInfo.showcasedTeams.length > 0 && (
-                    <Box style={{...globalStyles.flexBoxColumn, paddingBottom: globalMargins.small}}>
-                      <Box style={globalStyles.flexBoxRow}>
-                        <Text type="BodySmallSemibold">Teams:</Text>
+                {!loading && (
+                  <Box style={{...globalStyles.flexBoxColumn, paddingBottom: globalMargins.small}}>
+                    {(this.props.userInfo.showcasedTeams.length > 0 ||
```

Comment:
Profile code is pretty old/messy. We don't need to make this change now but in future stuff I'd advocate simpler logic in jsx and breaking up the components into sub components
```
const showEdit = (this.props.userInfo.showcasedTeams.length > 0 || (this.props.isYou && this.props.youAreInTeams)
```

```
{showEdit && <EditControl />}
```

Reply:
Could even pull these sub sections into their own components
```
{showEdit && <EditControl />}
```

Reply:
Cool, makes sense.  I'll merge this and open a new PR to clean it up.

Reply:
lets just ticket and leave in the backlog. i think we'll want to revisit this and merge desktop and native also